src/content/docs/agents/net-agent/attributes/custom-attributes-net.mdx

```diff
@@ -1,14 +1,17 @@
 ---
-title: Custom attributes (.NET)
+title: .NET custom attributes
 tags:
   - Agents
   - NET agent
   - Attributes
 metaDescription: 'APM''s .NET agent: how custom attribute values are processed and how they will appear in APM.'
```

Comment:
I don't understand why this is APM"s with a doublequote. Shouldn't that just be one apostrophe, or is that a problem with using possessive in a meta description? I like to avoid the problem entirely in metadescription by changing this to something like ".NET agent: how custom attribute values are processed and how they will appear in APM."

Reply:
@barbnewrelic , it looks like the two single quotes get rendered as the HTML apostrophe: `&#x27;` So, this should be fine.